src/lib/api/services/CircuitsService.ts

```diff
@@ -112,6 +115,29 @@ export class CircuitsService {
     });
   }
 
+  /**
```

Comment:
Are these delete methods new or just moved from somewhere else?

Reply:
I'm not entirely sure why these weren't being generated before, it must have been some consequence of switching the options on the API client generator. These aren't exposed through the SDK though and we don't use them internally yet either, so shouldn't make much of a difference.